cpp/oneapi/dal/backend/dispatcher_cpu.hpp

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+* Copyright 2014-2020 Intel Corporation
```

Comment:
2014?

Reply:
This is a problem for whole new part, headers are copy-pasted from the old one. It should be fixed in separate PR. I'll create it as there are many things need to be cleaned up.